### PR TITLE
Update cloudbeat to version 8.10.0

### PR DESCRIPTION
**Pull request by uri-weisman**
### Summary of your changes
Update cloudbeat to `8.10.0` after a branching out to `8.9.x`

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

**Comment by amirbenun** [comment]
@uri-weisman Can you please attach the guide you used to bump this version?

File: .mergify.yml [hunk 1]
```diff
@@ -219,3 +219,16 @@ pull_request_rules:
...
+  - name: backport patches to 8.9 branch
+    conditions:
+      - merged
+      - label=backport-v8.9.0
+    actions:
+      backport:
+        assignees:
+          - "{{ author }}"
+        branches:
+          - "8.8"
```
Comment (amirbenun) [inline .mergify.yml:231]: 8.9

Comment (uri-weisman) [reply .mergify.yml]: 🙏

**Comment by uri-weisman** [comment]
> @uri-weisman Can you please attach the guide you used to bump this version?

https://docs.elastic.dev/security-solution/cloud-security-posture/release/cloudbeat-release

**Review by oren-zohar — APPROVED** [review]

**Comment by oren-zohar** [comment]
@uri-weisman @gurevichdmitry Our helm chart script is trying to use 8.10.0-SNAPSHOT which was not released yet
image

**Comment by gurevichdmitry** [comment]
> @uri-weisman @gurevichdmitry Our helm chart script is trying to use 8.10.0-SNAPSHOT which was not released yet image

@oren-zohar, Exactly, we need to wait for the first appearance of version 8.10.0-SNAPSHOT, as Helm relies on an environment variable defined in hermit.hcl.

**Comment by uri-weisman** [comment]
> @uri-weisman @gurevichdmitry Our helm chart script is trying to use 8.10.0-SNAPSHOT which was not released yet image

I'll be fixed in a few hours, do we want to push it as is or to revert hermit version to `8.9`?

**Pull request merged**